src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -462,11 +465,13 @@ public void ConfigureServices(IServiceCollection services)
                     }
                 });
             });
-        }
+       }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         {
+
```

Comment:
Just check over the formatting in this file?

Reply:
got some comments that are out of alignment and some empty lines

Reply:
Ta, I think some formatting went awry when I used Rider

Reply:
I've just checked this and I can still see problems with the formatting in the diff shown in Github. When I reformat with Rider there are differences.

Maybe try reformatting with Rider again?